### ISSUE TITLE
API site down

### ISSUE THREAD

**Issue by CarlosCD** [matthutchinson/acts_as_textcaptcha#22]
As today (Dec. 10, 2015), the textcaptcha.com API site seems to be down. It was working well yesterday.

    This site is currently unavailable, please try again later.

Best,
- C.

**Comment by matthutchinson** [comment]
Hi there, i'm afraid I'm not responsible for that site, only this gem.  @robtuley runs the API web servers.  Here's some contact info I found on the web;

* http://whois.stsoftware.biz/textcaptcha-com.htm
* http://engineering.rainchasers.com
* https://github.com/robtuley

**Comment by CarlosCD** [comment]
Awesome. Thanks!
The site is up now.

**Comment by robtuley** [comment]
Yeah my bad, had a etcd cluster failure which took quite a while to recover. I'm currently running at reduced resilience in an effort to restore normal service as quickly as possible, hopefully back to normal soon. Worth being aware you may see some momentarily failures as I make the required changes probably tomorrow am now.

**Comment by CarlosCD** [comment]
Oh, thanks for the heads' up!

Murphy's law... It just happened that I set up a web form in production the day before for NASA (at hubble25th.org)... not a big deal, the form has not been advertised yet, and I saw the problem before they did.  Now I added a few fallback questions just in case (I should have done this in the first place...),  so all will be ok now, even if the API has any failure in the future.

Thanks again for the API. It is great!
